Mathlib/LinearAlgebra/Matrix/Charpoly/Coeff.lean

```diff
@@ -259,3 +260,36 @@ theorem coeff_charpoly_mem_ideal_pow {I : Ideal R} (h : ∀ i j, M i j ∈ I) (k
 #align coeff_charpoly_mem_ideal_pow coeff_charpoly_mem_ideal_pow
 
 end Ideal
+
+section reverse
+
+local notation:9000 R "[T;T⁻¹]" => LaurentPolynomial R
```

Comment:
Can you make this `scoped` notation in the Laurent files? (either in this PR, or a spin-off if you'd prefer)